### PR TITLE
Feature/dashboard card haptic feedback

### LINKED ISSUE #139: Lack of haptic feedback when long-pressing on a dashboard card
**Describe the bug**
The user should be able to feel when a card has been long pressed in the dashboard, but instead we only see the shadow indicating it can be moved. The shadow itself is harder to see when the dark theme is used in the app.

**To Reproduce**
1. Go to the Dashboard
2. Long press on any card

**Expected behavior**
That a haptic feedback is present alongside the shadow when long-pressing on a card

**Screenshots**
None

**Smartphone (please complete the following information):**
- Device: Pixel 3a
- OS: Android 11
- Browser: Chrome Beta
- Version: 91

**Additional context**
None

### PR DESCRIPTION
**Pull request by eneiss**
Closes #139 .
This might not work on all devices, see [the related issue on flutter repository](https://github.com/flutter/flutter/issues/73987).
(It did not work on my Samsung Galaxy A41, but it did on a S20 FE).

By the way I'm not sure if I put the new `haptics_container.dart` file at the right place, don't hesitate to tell me where I should move it if needed.

### CODE REVIEW

File: lib/ui/widgets/haptics_container.dart [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// FLUTTER / DART / THIRD-PARTIES
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+class HapticsContainer extends StatefulWidget {
+  final Widget content;
```
Comment (apomalyn) [inline lib/ui/widgets/haptics_container.dart:6]:
```suggestion
  final Widget child;
```

To respect the flutter name convention 😄 Could you apply this change everywhere? 😄

File: lib/ui/widgets/haptics_container.dart [hunk 2]
```diff
@@ -0,0 +1,25 @@
+// FLUTTER / DART / THIRD-PARTIES
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+class HapticsContainer extends StatefulWidget {
```
Comment (apomalyn) [inline lib/ui/widgets/haptics_container.dart:5]: Could you add a little description of the widget

**Review by apomalyn — CHANGES_REQUESTED** [review]
The HapticsContainer is at the right place, good job 😄

File: lib/ui/widgets/haptics_container.dart (hunk 1 above)
Comment (eneiss) [reply lib/ui/widgets/haptics_container.dart]: Fixed, thanks for pointing it up!

File: lib/ui/widgets/haptics_container.dart (hunk 2 above)
Comment (eneiss) [reply lib/ui/widgets/haptics_container.dart:11]: I added one, is it ok this way?

File: lib/ui/widgets/haptics_container.dart [hunk 3]
```diff
@@ -0,0 +1,31 @@
+// FLUTTER / DART / THIRD-PARTIES
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+/// A wrapper widget to add haptic feedback on an existing widget.
+///
+/// This is used to add haptic feedback on dashboard cards when reordering them:
+/// when long-pressed, a new HapticsContainer widget holding the actual card
+/// widget is created (through ReorderableListView's proxyDecorator) and haptic
+/// feedback is generated.
```
Comment (apomalyn) [inline lib/ui/widgets/haptics_container.dart:10]:
```suggestion
/// A wrapper widget to add haptic feedback on an existing widget.
///
/// For example this is used to add haptic feedback on dashboard cards when reordering them:
/// when long-pressed, a new HapticsContainer widget holding the actual card
/// widget is created (through ReorderableListView's proxyDecorator) and haptic
/// feedback is generated.
```

**Review by AlexandreMarkus — APPROVED** [review]
LGTM!! 💯

**Review by apomalyn — APPROVED** [review]
LGTM! :tada:

**Pull request merged**